# French translation for sbackup
# Copyright (c) 2007 Rosetta Contributors and Canonical Ltd 2007
# This file is distributed under the same license as the sbackup package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2007.
#
msgid ""
msgstr ""
"Project-Id-Version: sbackup\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2010-08-15 16:39+0200\n"
"PO-Revision-Date: 2010-07-19 22:43+0000\n"
"Last-Translator: François Tissandier <francois.tissandier@gmail.com>\n"
"Language-Team: French <fr@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2010-07-21 03:39+0000\n"
"X-Generator: Launchpad (build Unknown)\n"

#: src/sbackup/backupproc.py:168 src/sbackup/pkginfo.py:63
#, fuzzy, python-format
msgid "SBackup Daemon <%(login)s@%(hostname)s>"
msgstr "Démon sbackup <%(login)s@%(hostname)s>"

#: src/sbackup/backupproc.py:172
#, fuzzy, python-format
msgid "[SBackup] [%(profile)s] Report of %(date)s"
msgstr "[sbackup] [%(profile)s] Rapport du %(date)s"

#: src/sbackup/backupproc.py:177 src/sbackup/core/profile_handler.py:279
msgid "No log file specified."
msgstr "Aucun fichier journal indiqué."

#: src/sbackup/backupproc.py:182
msgid "Unable to find log file."
msgstr "Impossible de trouver le fichier journal."

#: src/sbackup/backupproc.py:248
msgid ""
"Critical Error: No configuration file for the default profile was found!\n"
"\n"
"Now continue processing remaining profiles."
msgstr ""
"Erreur critique: Aucun fichier de configuration trouvé pour le profile par "
"défaut."

#: src/sbackup/backupproc.py:297
msgid "Backup was canceled by user."
msgstr ""

#: src/sbackup/backupproc.py:302
#, fuzzy
msgid "An error occurred during the cancellation of backup:"
msgstr ""
"Une erreur s'est produite durant la sauvegarde :\n"
"%s"

#: src/sbackup/backupproc.py:308
#, fuzzy
msgid "An error occurred during the backup:"
msgstr ""
"Une erreur s'est produite durant la sauvegarde :\n"
"%s"

#: src/sbackup/backupproc.py:325
msgid "Error when sending email:"
msgstr ""

#: src/sbackup/backupproc.py:356
msgid "The following error(s) occurred before:"
msgstr "Les érreurs suivantes ont eu lieu avant:"

#: src/sbackup/backupproc.py:535
#, fuzzy, python-format
msgid ""
"Backup is not being started.\n"
"%s"
msgstr "Le processus de sauvegarde est en cours de démarrage"

#: src/sbackup/pkginfo.py:43
msgid ""
"This is a user friendly backup solution for common desktop needs. If you "
"want to help by submitting bugs, by translating or anything else please "
"visit our website."
msgstr ""
"sbackup est une solution convivial de sauvegarde pour un PC de bureau. Si "
"vous souhaitez aider en soumettant des bugs, en traduisant ou autre, merci "
"de visiter notre site internet."

#: src/sbackup/pkginfo.py:55
msgid "translator-credits"
msgstr ""
"Launchpad Contributions:\n"
"  François Tissandier https://launchpad.net/~baloo\n"
"  Jean-Peer Lorenz https://launchpad.net/~peer.loz\n"
"  Johan Serre https://launchpad.net/~serre-johan\n"
"  Oumar Aziz OUATTARA https://launchpad.net/~wattazoum\n"
"  PirBoazo https://launchpad.net/~pierre-boizot-name\n"
"  duke05 https://launchpad.net/~presto-duke"

#: src/sbackup/ar_backend/tar.py:108 src/sbackup/ar_backend/tar.py:160
#: src/sbackup/ar_backend/tar.py:206
msgid "Invalid archive type."
msgstr ""

#: src/sbackup/ar_backend/tar.py:327
msgid ""
"For the moment split functionality is not compatible with compress option."
msgstr ""

#: src/sbackup/ar_backend/tar.py:336 src/sbackup/ar_backend/tar.py:395
#: src/sbackup/core/profile_handler.py:289
#, python-format
msgid "Unable to change permissions for file '%s'."
msgstr "Impossible de changer la permission du fichier '%s'."

#: src/sbackup/ar_backend/tar.py:345
#, fuzzy, python-format
msgid "Unable to remove temporary file `%(file)s`: %(error)s"
msgstr "Impossible de supprimer le fichier verrou : %s"

#: src/sbackup/ar_backend/tar.py:362
msgid "Launching TAR to make incremental backup."
msgstr ""

#: src/sbackup/ar_backend/tar.py:388
msgid "Unable to find the SNAR file to make an incremental backup."
msgstr ""

#: src/sbackup/ar_backend/tar.py:389
msgid "Falling back to full backup."
msgstr ""

#: src/sbackup/ar_backend/tar.py:431
msgid "Launching TAR to make a full backup."
msgstr ""

#: src/sbackup/ar_backend/tar.py:495
msgid "Leading '/' from member names were removed."
msgstr ""

#: src/sbackup/ar_backend/tar.py:497
msgid "Leading '/' from hard link targets were removed."
msgstr ""

#: src/sbackup/ar_backend/tar.py:506
#, python-format
msgid "TAR returned a message: %s"
msgstr ""

#: src/sbackup/ar_backend/tar.py:508
#, python-format
msgid ""
"TAR returned messages:\n"
"%s"
msgstr ""

#: src/sbackup/ar_backend/tar.py:511
msgid "TAR has been finished successfully."
msgstr ""

#: src/sbackup/ar_backend/tar.py:515
#, python-format
msgid "TAR returned a warning: %s"
msgstr ""

#: src/sbackup/ar_backend/tar.py:517
#, python-format
msgid ""
"TAR returned warnings:\n"
"%s"
msgstr ""

#: src/sbackup/ar_backend/tar.py:520
msgid "TAR returned warnings but has been finished successfully."
msgstr ""

#: src/sbackup/ar_backend/tar.py:524
msgid "Unable to finish successfully. TAR terminated with errors."
msgstr ""

#: src/sbackup/ar_backend/tar.py:705
#, fuzzy, python-format
msgid "Unable to terminate backend process: %s"
msgstr "Impossible de supprimer le fichier verrou : %s"

#: src/sbackup/ar_backend/tar.py:738
msgid "Included"
msgstr "Inclus"

#: src/sbackup/ar_backend/tar.py:738
msgid "Not changed"
msgstr "Non changé"

#: src/sbackup/ar_backend/tar.py:738
msgid "Directory"
msgstr "Répertoire"

#: src/sbackup/ar_backend/tar.py:766
msgid "Line must be a string"
msgstr "Line doit être une chaîne de caractères"

#: src/sbackup/ar_backend/tar.py:782
msgid "Dumpdir inconsistancy : 'filename' is empty"
msgstr "Inconsistence de 'Dumpdir' : 'filename' est vide"

#: src/sbackup/ar_backend/tar.py:795
msgid "Dumpdir inconsistancy : 'control' is empty"
msgstr "Inconsistence de 'Dumpdir' : 'control' est vide"

#: src/sbackup/ar_backend/tar.py:868
#, python-format
msgid "File '%s' does not exist."
msgstr ""

#: src/sbackup/ar_backend/tar.py:998 src/sbackup/ar_backend/tar.py:1032
#: src/sbackup/ar_backend/tar.py:1129
msgid "The snarfile header is incomplete."
msgstr ""

#: src/sbackup/ar_backend/tar.py:1298
msgid "A SnapshotFile is required"
msgstr "Un SnapshotFile est requis"

#: src/sbackup/ar_backend/tar.py:1398
#, python-format
msgid "Directory does not exist: %s."
msgstr ""

#: src/sbackup/core/ConfigManager.py:273
#, python-format
msgid "Profile settings are being read from file '%s'."
msgstr "Les réglages du profil sont lus à partir du fichier '%s'."

#: src/sbackup/core/ConfigManager.py:276
#, python-format
msgid ""
"Profile settings are being set to default values. Configuration file is set "
"to '%s'."
msgstr ""
"Les réglages du profil sont réglés par défaut. Le fichier de configuration "
"est '%s'."

#: src/sbackup/core/ConfigManager.py:422
msgid "Option 'target' is missing, aborting."
msgstr "L'option 'target' est absente. Abandon !"

#: src/sbackup/core/ConfigManager.py:524 src/sbackup/core/ConfigManager.py:558
#, python-format
msgid ""
"Unable to evaluate '%(parameter)s' as a dictionary (value got = '%(value)r')."
msgstr ""

#: src/sbackup/core/ConfigManager.py:550
#, python-format
msgid ""
"Couldn't evaluate '%(parameter)s' as a dictionary (value got = '%(value)r' )"
msgstr ""
"Impossible d'évaluer '%(parameter)s' comme un dictionnaire (valeur obtenue = "
"'%(value)r' )"

#: src/sbackup/core/ConfigManager.py:574
msgid "You must provide a dictionary."
msgstr "Vous devez fournir un dictionnaire."

#: src/sbackup/core/ConfigManager.py:691
#, python-format
msgid "Log output for [%(profile)s] is directed to file '%(file)s'."
msgstr "Le journal pour [%(profile)s] sera écrit dans '%(file)s'."

#: src/sbackup/core/ConfigManager.py:698
#, python-format
msgid "Log output for [%s] is not directed into a file."
msgstr "La journal pour [%s] n'est pas écrit dans un fichier."

#: src/sbackup/core/ConfigManager.py:717
#, python-format
msgid "Unable to rename log file '%(src)s'->'%(dst)s': %(err)s"
msgstr ""
"Impossible de renommer le fichier journal '%(src)s'->'%(dst)s': %(err)s"

#: src/sbackup/core/ConfigManager.py:737
#, python-format
msgid "The config file '%s' couldn't be read!"
msgstr "Le fichier de configuration '%s' n'a pas pu être lu !"

#: src/sbackup/core/ConfigManager.py:749
#, python-format
msgid "section [%(section)s] in '%(configfile)s' should not exist, aborting"
msgstr ""
"La section [%(section)s] dans '%(configfile)s' ne devrait pas exister. "
"Abandon."

#: src/sbackup/core/ConfigManager.py:815
msgid "No schedule defined in configuration file. Probing from filesystem."
msgstr ""
"Pas de calendrier défini dans le fichier de configuration. Recherche dans le "
"système de fichier."

#: src/sbackup/core/ConfigManager.py:909 src/sbackup/core/ConfigManager.py:921
msgid "The config file is not set yet into this ConfigManager"
msgstr ""
"Le fichier de configuration n'est pas encore précisé dans ce ConfigManager"

#: src/sbackup/core/ConfigManager.py:1081
msgid "No receiver set."
msgstr "Pas de destination défini."

#: src/sbackup/core/ConfigManager.py:1083
msgid "No SMTP server set."
msgstr "Pas de serveur SMTP défini."

#: src/sbackup/core/ConfigManager.py:1086
msgid "Username set but no password specified."
msgstr "Nom d'utilisateur défini, mais pas le mot de passe."

#: src/sbackup/core/ConfigManager.py:1089
msgid "Password set but no username specified."
msgstr "Mot de passe défini, mais pas le nom d'utilisateur."

#: src/sbackup/core/ConfigManager.py:1092
msgid ""
"A certificate and key file is given while SSL option (smtptls=1) is not "
"set.\n"
"Select SSL in order to use Certificate and Key."
msgstr ""
"Un certificat et un fichier clé sont fournis alors que l'option SSL "
"(smtptls=1) n'est pas activé.\n"
"Sélectionnez SSL afin d'utiliser le certificat et la clé."

#: src/sbackup/core/ConfigManager.py:1096
msgid ""
"When specifying a SSL certificate or key file, a key file resp. certificate "
"is mandatory."
msgstr ""

#: src/sbackup/core/ConfigManager.py:1287
msgid "Default Profile"
msgstr "Profil par défaut"

#: src/sbackup/core/ConfigManager.py:1288
msgid "Unknown Profile"
msgstr "Profil inconnu"

#: src/sbackup/core/ConfigManager.py:1338
msgid "Unlimited"
msgstr "Illimité"

#: src/sbackup/core/ConfigManager.py:1339
msgid "100 MiB"
msgstr "100 Mo"

#: src/sbackup/core/ConfigManager.py:1340
msgid "250 MiB"
msgstr "250 Mo"

#: src/sbackup/core/ConfigManager.py:1341
msgid "650 MiB"
msgstr "650 Mo"

#: src/sbackup/core/ConfigManager.py:1342
msgid "2 GiB (FAT16)"
msgstr "2 Go (FAT16)"

#: src/sbackup/core/ConfigManager.py:1343
msgid "4 GiB (FAT32)"
msgstr "4 Go (FAT32)"

#: src/sbackup/core/ConfigManager.py:1344 src/sbackup/ui/configgui.py:275
#: src/sbackup/ui/configgui.py:1386
msgid "Custom"
msgstr "Personnalisé"

#: src/sbackup/core/ConfigManager.py:1346
msgid "MP3 Music"
msgstr "Musique MP3"

#: src/sbackup/core/ConfigManager.py:1347
msgid "AVI Video"
msgstr "Vidéo AVI"

#: src/sbackup/core/ConfigManager.py:1348
#: src/sbackup/core/ConfigManager.py:1349
msgid "MPEG Video"
msgstr "Vidéo MPEG"

#: src/sbackup/core/ConfigManager.py:1350
msgid "Matrjoshka Video"
msgstr "Vidéo Matroska"

#: src/sbackup/core/ConfigManager.py:1351
msgid "OGG Multimedia container"
msgstr "Conteneur multimédia OGG"

#: src/sbackup/core/ConfigManager.py:1352
msgid "CD Images"
msgstr "Images CD"

#: src/sbackup/core/RestoreManager.py:79
#, python-format
msgid "File '%s' not found in the backup snapshot files list"
msgstr "Fichier '%s' non trouvé dans la liste de fichiers de la sauvegarde"

#: src/sbackup/core/RestoreManager.py:81
#, fuzzy, python-format
msgid ""
"File '%(filename)s' not found in snapshot's [%(snapshotname)s] files list, "
"Skipped."
msgstr ""
"Fichier '%(filename)s' non trouvé dans la liste de fichier de la sauvegarde "
"[%(snapshotname)s]. Nous n'allons cependant pas nous arréter"

#: src/sbackup/core/SnapshotManager.py:125
#, python-format
msgid "Snapshot '%s' not found "
msgstr "Sauvegarde '%s' non trouvée. "

#: src/sbackup/core/SnapshotManager.py:216
#, python-format
msgid "Invalid snapshot `%(name)s` found: Name of snapshot not valid."
msgstr ""

#: src/sbackup/core/SnapshotManager.py:219
#, fuzzy, python-format
msgid "Invalid snapshot `%(name)s` found: %(error_cause)s."
msgstr ""
"Sauvegarde '%(name)s' non valide. En cours de suppression : %(error_cause)s "

#: src/sbackup/core/SnapshotManager.py:309
#, python-format
msgid "No need to rebase a full snapshot '%s'"
msgstr ""
"Il n'est pas nécessaire de changer la base d'une sauvegarde complète '%s'"

#: src/sbackup/core/SnapshotManager.py:311
#, python-format
msgid ""
"'%(snapshotToRebase)s'  doesn't have 'base' file , it might have been broken "
msgstr ""
"%(snapshotToRebase)s' n'a pas le fichier 'base'. Il se pourrait qu'il soit "
"cassé. "

#: src/sbackup/core/SnapshotManager.py:316
#, python-format
msgid ""
"Cannot rebase a snapshot on an earlier one : '%(snapshotToRebase)s' <= '%"
"(NewBaseSnapshot)s' "
msgstr ""
"Nous ne pouvons pas changer la base d'une sauvegarde à une sauvegarde "
"postérieure : '%(snapshotToRebase)s' <= '%(NewBaseSnapshot)s' "

#: src/sbackup/core/SnapshotManager.py:323
#, python-format
msgid "Rebasing '%(current)s' on '%(base)s'"
msgstr "Changement de la base de '%(current)s' à '%(base)s'"

#: src/sbackup/core/SnapshotManager.py:560
#, python-format
msgid "An error occurred when pulling snapshot '%s'."
msgstr ""
"Une erreur s'est produite lors de la récupération de l'instantané '%s'."

#: src/sbackup/core/SnapshotManager.py:599
msgid "The snarfile header is incomplete !"
msgstr "L'entête du fichier SNAR est incomplète !"

#: src/sbackup/core/SnapshotManager.py:751
#, python-format
msgid "Base of snapshot '%s' is not set."
msgstr ""

#: src/sbackup/core/SnapshotManager.py:788
#, python-format
msgid "Snapshot '%s' is already Full, nothing to do (not changing it to full)."
msgstr "L'instantanée '%s' est déjà plein, abandon de l'opération."

#: src/sbackup/core/SnapshotManager.py:828
#, python-format
msgid "Cancelling pull of snapshot '%s'"
msgstr "Abandon de l'export du contenu de la sauvegarde '%s'"

#: src/sbackup/core/SnapshotManager.py:998
msgid ""
"Logarithmic purge is currently disabled. Please specify a cutoff purge "
"interval in your configuration instead."
msgstr ""

#: src/sbackup/core/UpgradeManager.py:172
#, python-format
msgid "Upgrading from v1.0 to v1.2: %s"
msgstr "Mise à jour de v1.0 à v1.2 : %s"

#: src/sbackup/core/UpgradeManager.py:189
#, python-format
msgid "Upgrading to v1.3: %s"
msgstr "Mise à jour à v1.3 : %s"

#: src/sbackup/core/UpgradeManager.py:214
#, python-format
msgid "Upgrading to v1.4: %s"
msgstr "Mise à jour à v1.4 : %s"

#: src/sbackup/core/UpgradeManager.py:228
#, python-format
msgid "Upgrading to v1.5: %s"
msgstr "Mise à jour à v1.5 : %s"

#: src/sbackup/core/UpgradeManager.py:282
#, python-format
msgid "The SNAR file already exist for snapshot '%s'. It is not overwritten."
msgstr ""

#: src/sbackup/core/profile_handler.py:90
#, fuzzy
msgid "Preparation of backup process"
msgstr "Début de la session de sauvegarde"

#: src/sbackup/core/profile_handler.py:134
#, fuzzy
msgid ""
"There are snapshots stored in outdated snapshot formats. Please upgrade them "
"using 'Simple Backup-Restoration' if you want to use them."
msgstr ""
"Ces instantanés sont enregistrés dans un ancien format. Merci de les mettre "
"à jour en utilisant '(Not So) Simple Backup-Restoration' si vous souhaitez "
"les utiliser."

#: src/sbackup/core/profile_handler.py:143
#, python-format
msgid "Snapshot '%(name)s' is being made."
msgstr "Instantané '%(name)s' en cours de création."

#: src/sbackup/core/profile_handler.py:151
#, python-format
msgid "Setting Base to '%(value)s'."
msgstr ""

#: src/sbackup/core/profile_handler.py:160
msgid "Setting packages File."
msgstr "Réglage des fichiers de sauvegarde."

#: src/sbackup/core/profile_handler.py:166
msgid "Problem when setting the packages list: "
msgstr ""

#: src/sbackup/core/profile_handler.py:170
msgid "Setting Excludes File."
msgstr "Réglage des exclusions de fichiers."

#: src/sbackup/core/profile_handler.py:179
#, fuzzy, python-format
msgid "Setting compression format to `%s`"
msgstr "Réglage du format de compression"

#: src/sbackup/core/profile_handler.py:186
#, python-format
msgid "Setting size of archive chunks to %s"
msgstr ""

#: src/sbackup/core/profile_handler.py:193
msgid "Option 'Follow symbolic links' is enabled."
msgstr "L'option 'suivre les liens symboliques' est activée."

#: src/sbackup/core/profile_handler.py:195
msgid "Option 'Follow symbolic links' is disabled."
msgstr "L'option 'Suivre les liens symboliques' est désactivée."

#: src/sbackup/core/profile_handler.py:204
#, fuzzy
msgid "Snapshot is being committed"
msgstr "Instantané '%(name)s' en cours de création."

#: src/sbackup/core/profile_handler.py:218
#, fuzzy, python-format
msgid "Error while purging old snapshots: %s"
msgstr ""
"Une erreur s'est produite lors de la récupération de l'instantané '%s'."

#: src/sbackup/core/profile_handler.py:220
msgid "Backup process finished."
msgstr "Le processus de sauvegarde est terminé."

#: src/sbackup/core/profile_handler.py:227
msgid "Inspect file system and collect file infos"
msgstr ""

#: src/sbackup/core/profile_handler.py:238
msgid "Summary of backup"
msgstr ""

#: src/sbackup/core/profile_handler.py:239
#, python-format
msgid "Number of directories: %s."
msgstr "Nombre de répertoires : %s."

#: src/sbackup/core/profile_handler.py:240
#, python-format
msgid "Total number of files: %s."
msgstr "Nombre total de fichiers : %s."

#: src/sbackup/core/profile_handler.py:241
#, python-format
msgid "Number of symlinks: %s."
msgstr "Nombre de liens symboliques : %s."

#: src/sbackup/core/profile_handler.py:242
#, python-format
msgid "Number of files included in snapshot: %s."
msgstr "Nombre de fichiers inclus dans l'instantanée : %s."

#: src/sbackup/core/profile_handler.py:243
#, python-format
msgid "Number of new files (also included): %s."
msgstr "Nombre de nouveaux fichiers (également inclus) : %s."

#: src/sbackup/core/profile_handler.py:244
#, python-format
msgid "Number of files skipped in incremental snapshot: %s."
msgstr "Nombre de fichiers ignorés dans l'instantanée incrémental : %s."

#: src/sbackup/core/profile_handler.py:245
#, python-format
msgid "Number of items forced to be excluded: %s."
msgstr "Nombre de fichiers exclus de force : %s."

#: src/sbackup/core/profile_handler.py:246
#, python-format
msgid "Number of items to be excluded by config: %s."
msgstr "Nombre de fichiers exclus selon la configuration : %s."

#: src/sbackup/core/profile_handler.py:247
#, python-format
msgid "Maximum free size required is '%s'."
msgstr "L'espace libre maximum requis est '%s'."

#: src/sbackup/core/profile_handler.py:253
#, python-format
msgid "Available disk size is '%s'."
msgstr "L'espace disque disponible est '%s'."

#: src/sbackup/core/profile_handler.py:255
#, python-format
msgid ""
"Not enough free space in the target directory for the planned backup (%"
"(freespace)s <= %(neededspace)s)."
msgstr ""
"Pas assez d'espace dans le répertoire de destination pour la sauvegarde "
"planifiée (%(freespace)s <= %(neededspace)s)."

#: src/sbackup/core/profile_handler.py:274
msgid "Unable to copy log. File access is not initialized."
msgstr ""

#: src/sbackup/core/profile_handler.py:292
#, fuzzy, python-format
msgid "Unable to copy log file: %s"
msgstr "Impossible de supprimer le fichier verrou : %s"

#: src/sbackup/core/profile_handler.py:294
msgid "Unable to find logfile to copy into snapshot."
msgstr ""
"Impossible de trouver le fichier de journal à copier dans l'instantanée."

#: src/sbackup/core/profile_handler.py:296
msgid "No snapshot to copy logfile."
msgstr "Aucune sauvegarde pour copier le fichier de rapport."

#: src/sbackup/core/profile_handler.py:303
#, fuzzy
msgid "Processing of profile was canceled on user request\n"
msgstr "Le traitement du profil est terminé."

#: src/sbackup/core/profile_handler.py:320
#, fuzzy
msgid "Processing of profile successfully finished (no errors)\n"
msgstr "Le traitement du profil est terminé."

#: src/sbackup/core/profile_handler.py:326
#, fuzzy, python-format
msgid "Processing of profile failed with error: %s\n"
msgstr "Le traitement du profil est terminé."

#: src/sbackup/core/profile_handler.py:337
#, fuzzy, python-format
msgid "Backup destination: %s"
msgstr "NSbackup Restauration"

#: src/sbackup/core/profile_handler.py:341
#, fuzzy
msgid "Unable to find destination directory."
msgstr "Impossible d'ouvrir le répertoire cible.\n"

#: src/sbackup/core/profile_handler.py:360
msgid "Unable to find destination directory even after retry."
msgstr ""

#: src/sbackup/core/profile_handler.py:361
#: src/sbackup/core/profile_handler.py:368
#, python-format
msgid "Target directory '%(target)s' does not exist."
msgstr "Le répertoire de destination '%(target)s' n'existe pas."

#: src/sbackup/core/profile_handler.py:374
#, fuzzy, python-format
msgid "Unable to access destination: %s"
msgstr "Impossible d'envoyer la notification."

#: src/sbackup/core/snapshot.py:129
msgid "Snapshot is inconsistent: __name is not set "
msgstr ""

#: src/sbackup/core/snapshot.py:140
#, python-format
msgid "Name of snapshot '%s' doesn't match requirement"
msgstr "Le nom de la sauvegarde '%s' ne correspond pas au schéma attendu."

#: src/sbackup/core/snapshot.py:234
msgid "Snapshot is inconsistent: __snapshotpath is not set "
msgstr ""

#: src/sbackup/core/snapshot.py:321
#, python-format
msgid ""
"The snapshot compression format is supposed to be '%s' but the corresponding "
"well named file wasn't found"
msgstr ""
"Le format de compression de la sauvegarde est censé être '%s' mais le "
"fichier correctement nommé n'a pas été trouvé."

#: src/sbackup/core/snapshot.py:345
#, python-format
msgid ""
"%(file)s doesn't contain valid value. Ignoring incomplete or non - backup "
"directory. "
msgstr ""

#: src/sbackup/core/snapshot.py:482
#, python-format
msgid "Name of Snapshot not valid : %s"
msgstr "Nom de sauvegarde non valid : %s"

#: src/sbackup/core/snapshot.py:500
#, python-format
msgid "Name of base not valid : %s"
msgstr "Nom du parent invalide : %s"

#: src/sbackup/core/snapshot.py:546
#, fuzzy, python-format
msgid "Invalid name of snapshot: %s"
msgstr "Abandon de l'export du contenu de la sauvegarde '%s'"

#: src/sbackup/core/snapshot.py:550
#, fuzzy
msgid "Unable to read mandatory file `ver`"
msgstr "Impossible de supprimer le fichier verrou : %s"

#: src/sbackup/core/snapshot.py:595
msgid "Base name must be set for incremental backup."
msgstr ""

#: src/sbackup/fs_backend/_fuse_fam.py:106
msgid "Initializing FUSE File Access Manager."
msgstr "Initialisation du gestionnaire d'accès aux fichiers FUSE."

#: src/sbackup/fs_backend/_fuse_fam.py:129
msgid "Terminating FUSE File Access Manager."
msgstr ""

#: src/sbackup/fs_backend/_fuse_utils.py:44
#: src/sbackup/fs_backend/_gio_utils.py:106
msgid "SFTP"
msgstr ""

#: src/sbackup/fs_backend/_fuse_utils.py:45
#: src/sbackup/fs_backend/_gio_utils.py:107
msgid "SSH"
msgstr ""

#: src/sbackup/fs_backend/_fuse_utils.py:46
#: src/sbackup/fs_backend/_gio_utils.py:108
msgid "FTP"
msgstr ""

#: src/sbackup/fs_backend/_gio_fam.py:96
#, fuzzy
msgid "Initializing GIO File Access Manager."
msgstr "Initialisation du gestionnaire d'accès aux fichiers FUSE."

#: src/sbackup/fs_backend/_gio_fam.py:110
#, fuzzy
msgid "Terminating GIO File Access Manager."
msgstr "Initialisation du gestionnaire d'accès aux fichiers FUSE."

#: src/sbackup/fs_backend/_gio_utils.py:109
msgid "NFS"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:187
msgid "Unable to get mount: path is not mounted"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:189
msgid "Unable to get mount: path not found when mounting (is probably local)"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:191
#: src/sbackup/fs_backend/_gio_utils.py:194
#, fuzzy, python-format
msgid "Unable to get mount: %s"
msgstr "Impossible de supprimer le fichier verrou : %s"

#: src/sbackup/fs_backend/_gio_utils.py:241
msgid "Unable to un-set mount flag: path is not stored"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:355
msgid "Path is already mounted."
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:378
msgid "Path is not mounted."
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:809
msgid "test specified path for existence using GIO"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:822
#, fuzzy
msgid "Create testdir"
msgstr "Création du répertoire cible '%s'"

#: src/sbackup/fs_backend/_gio_utils.py:827
msgid "Test testfile for existence"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:834
#, fuzzy
msgid "Create file"
msgstr "Profil par défaut"

#: src/sbackup/fs_backend/_gio_utils.py:841
msgid "Re-read test file"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:853
msgid "Remove file"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:855
#, fuzzy
msgid "Remove dir"
msgstr "supprimer"

#: src/sbackup/fuse_plugins/__init__.py:160
#, fuzzy, python-format
msgid "Unable to import plugin `%(plugin_name)s`: %(error_cause)s "
msgstr ""
"Impossible d'importer le module %(plugin_name)s ! Cause : %(error_cause)s "

#: src/sbackup/fuse_plugins/ftpFuseFAM.py:117
#, python-format
msgid ""
"Couldn't found external application 'curlftpfs' needed for handling of ftp "
"sites: %s"
msgstr ""
"Impossible de trouver l'application externe 'curlftpfs' nécessaire à la "
"gestion des destinations FTP: %s"

#: src/sbackup/fuse_plugins/ftpFuseFAM.py:127
#: src/sbackup/fuse_plugins/sftpFuseFAM.py:124
#, python-format
msgid "Couldn't mount '%(server)s' into '%(mountpoint)s' : %(error)s"
msgstr "Impossible de monter %(server)s' sur '%(mountpoint)s' : %(error)s"

#: src/sbackup/fuse_plugins/ftpFuseFAM.py:141
msgid "FTP schema is: ftp://user:pass@server/anything"
msgstr "Le schéma FTP est : ftp://utilisateur:motdepasse@serveur/repertoire"

#: src/sbackup/fuse_plugins/ftpFuseFAM.py:188
#, python-format
msgid ""
"Error matching the schema 'ftp://user:pass@server/anything' with '%s' (The "
"'/' after server is mandatory)"
msgstr ""
"Problème sur la correspondance du schéma 'ftp://user:pass@server/anything' "
"avec '%s' (le '/' après 'server' est obligatoire)"

#: src/sbackup/fuse_plugins/sftpFuseFAM.py:114
#, python-format
msgid ""
"Couldn't found external application 'curlftpfs' needed for handling of sftp "
"sites: %s"
msgstr ""
"Impossible de trouver l'application externe 'curlftpfs' nécessaire à la "
"gestion des sites sftp : %s"

#: src/sbackup/fuse_plugins/sftpFuseFAM.py:138
msgid "SFTP schema is like : sftp://user:pass@server/anything"
msgstr ""
"Le modèle SFTP est : sftp://utilisateur:mot_de_passe@serveur/suite_du_chemin"

#: src/sbackup/fuse_plugins/sftpFuseFAM.py:185
#, python-format
msgid ""
"Error matching the schema 'sftp://user:pass@server/anything' with '%s' (The "
"'/' after server is mandatory)"
msgstr ""
"Problème sur la correspondance du schéma 'sftp://user:pass@server/anything' "
"avec '%s' (le '/' après 'server' est obligatoire)"

#: src/sbackup/fuse_plugins/sshFuseFAM.py:81
#, python-format
msgid ""
"Error matching the schema 'ssh://user:pass@example.com/home/' with '%s' (The "
"'/' after server is mandatory)"
msgstr ""
"Problème sur la correspondance du schéma 'ssh://user:pass@example.com/home/' "
"avec '%s' (le '/' après 'server' est obligatoire)"

#: src/sbackup/fuse_plugins/sshFuseFAM.py:137
#, python-format
msgid ""
"The sshfs command '%(command)s' didn't perform normally. Output => %"
"(erroroutput)s "
msgstr ""
"La commande sshfs '%(command)s' ne s'est pas exécutée normalement. Résultat "
"=> %(erroroutput)s "

#: src/sbackup/fuse_plugins/sshFuseFAM.py:142
msgid "SSH schema is like : ssh://user:pass@example.com:33/home/user/backup/"
msgstr ""
"Le schéma SSH est le suivant : ssh://utilisateur:motdepasse@serveur.com:33/"
"home/user/backup/"

#: src/sbackup/fuse_plugins/sshFuseFAM.py:158
#, python-format
msgid ""
"Error matching the schema 'ssh://user:pass@example.com:21/home/' with '%"
"s' (The '/' after server is mandatory)"
msgstr ""
"Impossible de faire correspondre le schéma 'ssh://utilisateur:"
"motdepasse@serveur.com:21/home/' avec '%s' (Le '/' après le nom du serveur "
"est obligatoire)"

#: src/sbackup/ui/GladeWindow.py:201
#, fuzzy
msgid ""
"Please make sure the missing directory exists (e.g. by mounting an external "
"disk) or change the specified target."
msgstr ""
"Assurez vous s'il vous plait que le répertoire manquant existe (par exemple "
"en montant un disque externe) ou en changer le répertoire de destination à "
"un répertoire existant."

#: src/sbackup/ui/GladeWindow.py:202
#, python-format
msgid ""
"While accessing the backup destination following error occurred:\n"
"\n"
"%s."
msgstr ""

#: src/sbackup/ui/GladeWindow.py:203 src/sbackup/ui/restoregui.py:783
#: src/sbackup/ui/restoregui.py:837
#, fuzzy
msgid "Simple Backup error"
msgstr "Erreur (Not So) Simple Backup"

#: src/sbackup/ui/GladeWindow.py:204
#, fuzzy
msgid "Unable to access backup destination"
msgstr "Impossible d'ouvrir la destination de sauvegarde."

#: src/sbackup/ui/configgui.py:121
msgid "Scheduled backups are available for Administrator users only."
msgstr ""
"Les sauvegardes programmées sont disponibles seulement pour les "
"administrateurs."

#: src/sbackup/ui/configgui.py:131 src/sbackup/ui/configgui.py:140
msgid "Name"
msgstr "Nom"

#: src/sbackup/ui/configgui.py:148
msgid "File Type"
msgstr "Type de Fichier"

#: src/sbackup/ui/configgui.py:183
msgid "Enable"
msgstr "Activé"

#: src/sbackup/ui/configgui.py:184
msgid "Profile Name"
msgstr "Nom du profil"

#: src/sbackup/ui/configgui.py:281
#, python-format
msgid ""
"Invalid or empty regular expression ('%s') found in configuration. Removed."
msgstr ""
"Une expression régulière invalide ou vide ('%s') a été trouvée dans la "
"configuration et supprimée."

#: src/sbackup/ui/configgui.py:288
#, python-format
msgid ""
"Invalid or empty regular expressions found\n"
"in configuration file:\n"
"'%s'\n"
"\n"
"These expressions are not used and were\n"
"removed from the configuration."
msgstr ""
"Des expressions régulières invalides ou vides ont été trouvées\n"
"dans le fichier de configuration :\n"
"'%s'\n"
"\n"
"Ces expressions ne sont pas utilisées et ont été\n"
"supprimées de la configuration."

#: src/sbackup/ui/configgui.py:535
msgid ""
"Please make sure the missing directory exists (e.g. by mounting an external "
"disk) or change the specified target to an existing one."
msgstr ""
"Assurez vous s'il vous plait que le répertoire manquant existe (par exemple "
"en montant un disque externe) ou en changer le répertoire de destination à "
"un répertoire existant."

#: src/sbackup/ui/configgui.py:536
msgid ""
"Backup target does not exist.\n"
"\n"
"Attention: The target will be set to the default value. Check this on the "
"destination settings page before saving the configuration."
msgstr ""
"La destination de sauvegarde n'existe pas.\n"
"\n"
"Attention : la destination va être réglé à la valeur par défaut. Merci de "
"vérifier ce réglage sur l'écran de définition de la destination avant de "
"sauvegarder la configuration."

#: src/sbackup/ui/configgui.py:538
msgid "Unable to open backup target"
msgstr "Impossible d'ouvrir la destination de sauvegarde."

#: src/sbackup/ui/configgui.py:609
#, python-format
msgid "Current profile: %s"
msgstr "Profile courant : %s"

#: src/sbackup/ui/configgui.py:700
msgid "Already included item !"
msgstr "Élément déjà ajouté !"

#: src/sbackup/ui/configgui.py:715
msgid "Already excluded item !"
msgstr "Élément déjà enlevé !"

#: src/sbackup/ui/configgui.py:727 src/sbackup/ui/configgui.py:1426
msgid "Empty expression. Please enter a valid regular expression."
msgstr "Expression vide. Veuillez entrer une expression régulière valide."

#: src/sbackup/ui/configgui.py:742 src/sbackup/ui/configgui.py:1440
msgid "Provided regular expression is not valid."
msgstr "L'expression régulière fournie est invalide."

#: src/sbackup/ui/configgui.py:750
msgid "Empty filename or path. Please enter a valid filename or path."
msgstr ""
"Nom de fichier ou chemin vide. Veuillez entrer un nom de fichier ou un "
"chemin valide."

#: src/sbackup/ui/configgui.py:759
msgid ""
"It seems the path you entered does not exists. Do you want to add this "
"incorrect path?"
msgstr ""
"Le chemin que vous avez entré ne semble pas exister. Voulez-vous tout de "
"même le conserver ?"

#: src/sbackup/ui/configgui.py:788
msgid "Save configuration as..."
msgstr "Sauver la configuration en tant que ..."

#: src/sbackup/ui/configgui.py:855
msgid ""
"A backup process is now executed in the background.\n"
"\n"
"You can monitor the progress of the backup by means of the status indicator "
"displayed in the notification area."
msgstr ""

#: src/sbackup/ui/configgui.py:856
#, fuzzy
msgid "Backup process started"
msgstr "Le processus de sauvegarde est en cours de démarrage"

#: src/sbackup/ui/configgui.py:861 src/sbackup/ui/configgui.py:1629
#: src/sbackup/ui/configgui.py:1999
#, python-format
msgid "Unexpected dialog response: %s"
msgstr ""

#: src/sbackup/ui/configgui.py:935
msgid "Include file..."
msgstr "Inclure le fichier ..."

#: src/sbackup/ui/configgui.py:942 src/sbackup/ui/configgui.py:997
msgid "All files"
msgstr "Tous fichiers"

#: src/sbackup/ui/configgui.py:962
msgid "Include folder..."
msgstr "Inclure le répertoire ..."

#: src/sbackup/ui/configgui.py:990
msgid "Exclude file..."
msgstr "Excluse le fichier ..."

#: src/sbackup/ui/configgui.py:1013
msgid "Exclude folder..."
msgstr "Exclure le répertoire ..."

#: src/sbackup/ui/configgui.py:1278
msgid "Configuration has changed"
msgstr "La configuration a été modifiée."

#: src/sbackup/ui/configgui.py:1279
msgid ""
"There are unsaved modifications. Please save the configuration or revert "
"these changes before testing the mail settings."
msgstr ""
"Il y a des modifications non sauvegardées. Merci de sauver la configuration "
"ou de réinitialiser ces changements avant de tester les réglages d\"email."

#: src/sbackup/ui/configgui.py:1285 src/sbackup/ui/configgui.py:1291
msgid "Test mail settings"
msgstr "Tester le paramétrage mail"

#: src/sbackup/ui/configgui.py:1286
msgid "The test failed with following output:"
msgstr "Le test a échoué avec le résultat suivant :"

#: src/sbackup/ui/configgui.py:1292
msgid "The test was successful."
msgstr "Le test est réussi."

#: src/sbackup/ui/configgui.py:1375 src/sbackup/ui/configgui.py:1418
#, python-format
msgid ""
"The given expression contains unsupported characters ('%s'). Currently it is "
"not possible to use these characters in exclude expressions."
msgstr ""
"L'expression fournie contient des caractères non supportés ('%s'). Il n'est "
"actuellement pas possible d'utiliser ces caractères dans les expressions "
"d'exclusion."

#: src/sbackup/ui/configgui.py:1378 src/sbackup/ui/configgui.py:1421
msgid "Unsupported character"
msgstr "Caractère non supporté."

#: src/sbackup/ui/configgui.py:1589
#, fuzzy
msgid "Connect to remote destination"
msgstr "Choisir le répertoire de destination"

#: src/sbackup/ui/configgui.py:1637
msgid "Mount was sucessful (no errors)"
msgstr ""

#: src/sbackup/ui/configgui.py:1648
#, fuzzy
msgid "Unable to mount host"
msgstr "Echec de la mise à jour de la sauvegarde."

#: src/sbackup/ui/configgui.py:1657
msgid "Umount was sucessful (no errors)"
msgstr ""

#: src/sbackup/ui/configgui.py:1667
#, fuzzy
msgid "Unable to unmount host"
msgstr "Echec de la mise à jour de la sauvegarde."

#: src/sbackup/ui/configgui.py:1697
msgid "Perfom tests on remote host"
msgstr ""

#: src/sbackup/ui/configgui.py:1710
#, fuzzy
msgid "Unable to access remote destination"
msgstr "Impossible d'envoyer la notification."

#: src/sbackup/ui/configgui.py:1714
msgid "All tests passed"
msgstr ""

#: src/sbackup/ui/configgui.py:1871
msgid ""
"The given name of the new profile is empty. Please enter a valid profile "
"name."
msgstr ""
"Le nom fourni pour le nouveau profile est vide. Merci d'entrer un nom de "
"profil correct."

#: src/sbackup/ui/configgui.py:1872 src/sbackup/ui/configgui.py:1879
msgid "Profile name not valid"
msgstr "Le nom de profil n'est pas valide."

#: src/sbackup/ui/configgui.py:1878
msgid ""
"The given name of the new profile already exists. Please enter another name."
msgstr ""
"Le nom donné au nouveau profil existe déjà. Merci d'entrer un autre nom."

#: src/sbackup/ui/configgui.py:1880
msgid "Renaming of profiles is not supported."
msgstr "Le changement de nom des profils n'est pas possible."

#: src/sbackup/ui/configgui.py:1910
#, fuzzy, python-format
msgid ""
"<b>Delete configuration profile?</b>\n"
"\n"
"You are trying to remove a configuration profile. You will not be able to "
"restore it. If you are not sure, use the 'enable|disable' functionality "
"instead.\n"
"\n"
"Do you really want to delete the profile '%(name)s'?"
msgstr ""
"Vous essayez de supprimer un profile. Vous ne pourrez pas le restaurer. \n"
" Si vous n'êtes pas sûr de ce que vous faites, merci d'utiliser la fonction "
"\"activé|désactivé'. \n"
"<b>Etes-vous sûr de vouloir supprimer le profil '%(name)s' ?</b>"

#: src/sbackup/ui/configgui.py:1983
#, python-format
msgid ""
"<big><b>Set default values for current profile?</b></big>\n"
"This will restore the default values for the profile currently edited: '%"
"s'.\n"
"\n"
"These predefined settings are recommended for most users. Check whether they "
"are appropriate for your use before saving the changed configuration."
msgstr ""
"<big><b>Activer les réglages par défaut pour le profil courant ?</b></big>\n"
"Cette opération va restaurer les réglages par défaut pour le profil en cours "
"d'édition: '%s'.\n"
"\n"
"Ces réglages par défaut sont recommandés pour la plupart des utilisateurs. "
"Merci de vérifier s'ils conviennent à votre usage avant de sauvegarder la "
"configuration."

#: src/sbackup/ui/configgui.py:1993
#, fuzzy
msgid "Default settings are being applied."
msgstr "Les réglages du profil sont lus à partir du fichier '%s'."

#: src/sbackup/ui/configgui.py:2018
msgid ""
"<b>Unable to remove default profile</b>\n"
"\n"
"The default profile cannot be removed. In the case you want to use just a "
"single profile, please set up the default profile accordingly."
msgstr ""

#: src/sbackup/ui/configgui.py:2032
msgid ""
"<b>Unable to remove default profile</b>\n"
"\n"
"The default profile cannot be disabled. In the case you want to use just a "
"single profile, please set up the default profile accordingly."
msgstr ""

#: src/sbackup/ui/configgui.py:2045
#, fuzzy
msgid "Please select a profile."
msgstr "Choisissez un profil, s'il vous plait"

#: src/sbackup/ui/indicator_core.py:116
#, fuzzy
msgid "there was a problem initializing the pynotify module"
msgstr "Il y a eu un problème en initialisant le "

#: src/sbackup/ui/indicator_core.py:145 src/sbackup/ui/indicator_core.py:192
#: src/sbackup/ui/indicator_core.py:220 src/sbackup/ui/indicator_core.py:242
#, fuzzy, python-format
msgid "Connection to notification-daemon failed: %s"
msgstr "La connexion au démon de notification a échoué : %s"

#: src/sbackup/ui/indicator_core.py:305
#, fuzzy
msgid "Simple Backup"
msgstr "Erreur (Not So) Simple Backup"

#: src/sbackup/ui/indicator_core.py:311
msgid "Show message windows"
msgstr ""

#: src/sbackup/ui/indicator_core.py:353 src/sbackup/ui/indicator_core.py:807
#, fuzzy
msgid "Cancel Backup"
msgstr "Nettoyage des anciennes sauvegardes"

#: src/sbackup/ui/indicator_core.py:415
msgid "Connection to D-Bus service lost."
msgstr ""

#: src/sbackup/ui/indicator_core.py:440
#, fuzzy
msgid "re-connection was successful"
msgstr "Le test est réussi."

#: src/sbackup/ui/indicator_core.py:479
msgid ""
"No backup in progress and connection to D-Bus service lost. Simple Backup "
"Indicator is being terminated."
msgstr ""

#: src/sbackup/ui/indicator_core.py:508
#, fuzzy
msgid "Starting backup session"
msgstr "Début de la session de sauvegarde"

#: src/sbackup/ui/indicator_core.py:522
#, fuzzy
msgid "Ending Backup Session"
msgstr "Début de la session de sauvegarde"

#: src/sbackup/ui/indicator_core.py:535
#, fuzzy
msgid ""
"There are snapshots with old snapshot format. Please upgrade these using the "
"Restoration tool if you want to use them."
msgstr ""
"Ces instantanés sont enregistrés dans un ancien format. Merci de les mettre "
"à jour en utilisant '(Not So) Simple Backup-Restoration' si vous souhaitez "
"les utiliser."

#: src/sbackup/ui/indicator_core.py:537
#, python-format
msgid "Unknown D-Bus event `%s` received."
msgstr ""

#: src/sbackup/ui/indicator_core.py:578
msgid "Try again"
msgstr ""

#: src/sbackup/ui/indicator_core.py:642
#, fuzzy
msgid ""
"Attempt of starting another instance of Simple Backup while this one is "
"already running."
msgstr ""
"Une autre instance de '(Not So) Simple Backup' est déjà en cours d'exécution "
"(processus id: %s)."

#: src/sbackup/ui/indicator_core.py:652
#, fuzzy, python-format
msgid "Unable to present window: %s"
msgstr "Impossible de supprimer le fichier verrou : %s"

#: src/sbackup/ui/indicator_core.py:668
#, fuzzy, python-format
msgid "Do you really want to cancel the backup process of profile '%s'?"
msgstr "Voulez vous vraiment changer la base de '%(current)s' à '%(base)s' ?"

#: src/sbackup/ui/indicator_core.py:672
#, fuzzy
msgid "Cancel running backup process?"
msgstr "Début de la session de sauvegarde"

#: src/sbackup/ui/indicator_core.py:689
#, fuzzy
msgid "Unable to cancel backup process."
msgstr "Impossible d'ouvrir la destination de sauvegarde."

#: src/sbackup/ui/indicator_core.py:751
#, fuzzy, python-format
msgid "Profile: %s"
msgstr "Profils"

#: src/sbackup/ui/indicator_core.py:752
#, python-format
msgid "Size of backup: %s"
msgstr ""

#: src/sbackup/ui/indicator_core.py:753
#, python-format
msgid "%.1f%% processed"
msgstr ""

#: src/sbackup/ui/indicator_core.py:754
#, python-format
msgid "Remaining time: %s"
msgstr ""

#: src/sbackup/ui/indicator_core.py:800
#, fuzzy, python-format
msgid "PID of backup process being canceled: %s"
msgstr "Le processus de sauvegarde est en cours de démarrage"

#: src/sbackup/ui/indicator_core.py:805
#, fuzzy, python-format
msgid "Unable to send signal to process %s"
msgstr "Impossible d'envoyer la notification."

#: src/sbackup/ui/indicator_core.py:806
#, python-format
msgid "Sending signal using gksu to process %s"
msgstr ""

#: src/sbackup/ui/indicator_core.py:810
#, python-format
msgid "Sending signal using gksu to process %s failed"
msgstr ""

#: src/sbackup/ui/indicator_core.py:845
#, python-format
msgid ""
"<b>Unable to find specified target directory</b>\n"
"\n"
"The specified target directory '%s' was not found.\n"
"\n"
msgstr ""

#: src/sbackup/ui/indicator_core.py:846
#, python-format
msgid ""
"You can try to use the specified target again or cancel the profile "
"execution. The profile execution is canceled automatically in %s seconds."
msgstr ""

#: src/sbackup/ui/indicator_core.py:872 src/sbackup/ui/indicator_core.py:914
#: src/sbackup/ui/indicator_core.py:932 src/sbackup/ui/indicator_core.py:940
#: src/sbackup/ui/indicator_core.py:957
msgid "unknown"
msgstr ""

#: src/sbackup/ui/indicator_core.py:893
msgid "less than 1 minute"
msgstr ""

#: src/sbackup/ui/indicator_core.py:895
msgid "about 2 minutes"
msgstr ""

#: src/sbackup/ui/indicator_core.py:897
msgid "about 3 minutes"
msgstr ""

#: src/sbackup/ui/indicator_core.py:899
msgid "about 4 minutes"
msgstr ""

#: src/sbackup/ui/indicator_core.py:901
msgid "about 5 minutes"
msgstr ""

#: src/sbackup/ui/indicator_core.py:903
#, python-format
msgid "about %.0f minutes"
msgstr ""

#: src/sbackup/ui/indicator_core.py:913
msgid "In progress"
msgstr ""

#: src/sbackup/ui/indicator_core.py:923 src/sbackup/ui/indicator_core.py:949
msgid "finished"
msgstr ""

#: src/sbackup/ui/indicator_core.py:931
msgid "Progress: unknown"
msgstr ""

#: src/sbackup/ui/indicator_core.py:938
#, fuzzy
msgid "Backup is being prepared"
msgstr "Le processus de sauvegarde est en cours de démarrage"

#: src/sbackup/ui/indicator_core.py:939
msgid "Collecting file info"
msgstr ""

#: src/sbackup/ui/indicator_core.py:948
msgid "Progress: canceled"
msgstr ""

#: src/sbackup/ui/indicator_core.py:968
msgid "An error occurred"
msgstr ""

#: src/sbackup/ui/indicator_core.py:1001
#, fuzzy
msgid "Another `Simple Backup Indicator` is already running."
msgstr ""
"Une autre instance de '(Not So) Simple Backup' est déjà en cours d'exécution "
"(processus id: %s)."

#: src/sbackup/ui/indicator_core.py:1025
#, fuzzy, python-format
msgid "Initialization of GTK+ failed: %s"
msgstr "L'initialisation du module 'pynotify' a échoué"

#: src/sbackup/ui/misc.py:293
#, python-format
msgid "%s   (Administrator mode)"
msgstr "%s   (mode administrateur)"

#: src/sbackup/ui/misc.py:323
#, python-format
msgid ""
"An uncaught error occurred. Close this message window and restart the "
"application.\n"
"\n"
"Please report this error on https://bugs.launchpad.net/sbackup.\n"
"\n"
"%s"
msgstr ""

#: src/sbackup/ui/misc.py:324
msgid "Sorry, this should not have happened"
msgstr ""

#: src/sbackup/ui/restoregui.py:77
msgid "Restore..."
msgstr "Restoration ..."

#: src/sbackup/ui/restoregui.py:78 data/ui/sbackup-restore.glade:845
msgid "Restore as..."
msgstr "Restoration en tant que ..."

#: src/sbackup/ui/restoregui.py:79
msgid "Revert..."
msgstr "Rétrograder ..."

#: src/sbackup/ui/restoregui.py:80 data/ui/sbackup-restore.glade:877
msgid "Revert as..."
msgstr "Rétrograder comme ..."

#: src/sbackup/ui/restoregui.py:127 data/ui/sbackup-restore.glade:111
#: data/ui/sbackup-restore.glade:591
msgid "n.a."
msgstr ""

#: src/sbackup/ui/restoregui.py:145 src/sbackup/ui/restoregui.py:162
msgid "Snapshots"
msgstr "Sauvegardes"

#: src/sbackup/ui/restoregui.py:146
msgid "Version"
msgstr "Version"

#: src/sbackup/ui/restoregui.py:153 data/ui/sbackup-config.glade:4063
msgid "Path"
msgstr "Chemin"

#: src/sbackup/ui/restoregui.py:158
msgid "State"
msgstr "État"

#: src/sbackup/ui/restoregui.py:177
#, fuzzy
msgid "Connect to destination..."
msgstr "Choisir le répertoire de destination"

#: src/sbackup/ui/restoregui.py:201
#, fuzzy
msgid "Specified path does not exist"
msgstr "Le fichier '%(file)s' n'existe pas."

#: src/sbackup/ui/restoregui.py:246
msgid "Ready"
msgstr "Prêt"

#: src/sbackup/ui/restoregui.py:360
#, fuzzy
msgid "No destination specified"
msgstr "Aucun fichier journal indiqué."

#: src/sbackup/ui/restoregui.py:377
msgid "Choose a source folder"
msgstr "Choisissez un dossier source"

#: src/sbackup/ui/restoregui.py:394
#, fuzzy
msgid "Reading snapshots..."
msgstr "Lecture de la sauvegarde ..."

#: src/sbackup/ui/restoregui.py:445 src/sbackup/ui/restoregui.py:483
msgid "Loading ..."
msgstr "Chargement en cours..."

#: src/sbackup/ui/restoregui.py:532
#, fuzzy
msgid "Simple Backup restore error"
msgstr "Erreur de restauration (Not So) Simple Backup"

#: src/sbackup/ui/restoregui.py:533
msgid "An error occurred while reading snapshot"
msgstr "Une erreur s'est produite pendant la lecture de l'instantané."

#: src/sbackup/ui/restoregui.py:538
msgid "This snapshot seems empty."
msgstr "Cette sauvegarde semble vide."

#: src/sbackup/ui/restoregui.py:555
msgid "Reading backup snapshot..."
msgstr "Lecture de la sauvegarde ..."

#: src/sbackup/ui/restoregui.py:578
msgid "No backups found for this day."
msgstr "Pas de sauvegarde trouvée pour ce jour."

#: src/sbackup/ui/restoregui.py:593
#, fuzzy, python-format
msgid ""
"Snapshot version is not supported (Only %(supportedversion)s is supported). "
"Version '%(currentversion)s' found. You should upgrade it. "
msgstr ""
"La version de la sauvegarde n'est pas supportée (seule '%(supportedversion)"
"s' est supportée). Version '%(currentversion)s' trouvée. Vous devriez mettre "
"à jour la sauvegarde. "

#: src/sbackup/ui/restoregui.py:620
msgid "Select restore location"
msgstr "Sélectionnez l'emplacement de restauration"

#: src/sbackup/ui/restoregui.py:638
#, python-format
msgid "Do you really want to revert '%s'?"
msgstr ""

#: src/sbackup/ui/restoregui.py:651
msgid "Select revert location"
msgstr "Choisissez la destionnation de restauration"

#: src/sbackup/ui/restoregui.py:660
#, python-format
msgid "Do you really want to revert '%(source)s' to '%(dir)s'?"
msgstr ""

#: src/sbackup/ui/restoregui.py:738
#, fuzzy
msgid "Reading snapshot..."
msgstr "Lecture de la sauvegarde ..."

#: src/sbackup/ui/restoregui.py:757
msgid "The version of the snapshot is greater than the supported one!"
msgstr "La version de la sauvegarde est plus haute que celle supportée."

#: src/sbackup/ui/restoregui.py:767
#, python-format
msgid ""
"Upgrading the selected snapshot `%s` might corrupt your backup for this "
"particular point in time. You cannot undo this operation. It is recommended "
"to create a copy of the snapshot directory before upgrading it. The upgrade "
"process takes a while - be patient.\n"
"\n"
"Are you sure that you want to upgrade the snapshot now?"
msgstr ""

#: src/sbackup/ui/restoregui.py:771
#, fuzzy
msgid "Do you really want to upgrade snapshot?"
msgstr "Echec de la mise à jour de la sauvegarde."

#: src/sbackup/ui/restoregui.py:781
#, python-format
msgid ""
"While attempting to upgrade snapshot the following error occurred:\n"
"%s"
msgstr ""

#: src/sbackup/ui/restoregui.py:784
msgid "Unable to upgrade snapshot"
msgstr "Echec de la mise à jour de la sauvegarde."

#: src/sbackup/ui/restoregui.py:804
#, python-format
msgid "Do you really want to rebase '%(current)s' on '%(base)s' ?"
msgstr "Voulez vous vraiment changer la base de '%(current)s' à '%(base)s' ?"

#: src/sbackup/ui/restoregui.py:822
#, python-format
msgid ""
"Deleting the selected snapshot `%s` will remove your backup for this "
"particular point in time. You cannot undo this operation.\n"
"\n"
"Are you sure that you want to remove the snapshot permanently?"
msgstr ""

#: src/sbackup/ui/restoregui.py:826
#, fuzzy
msgid "Do you really want to remove snapshot?"
msgstr "Voulez vous vraiment changer la base de '%(current)s' à '%(base)s' ?"

#: src/sbackup/ui/restoregui.py:835
#, python-format
msgid ""
"While attempting to delete snapshot the following error occurred:\n"
"%s"
msgstr ""

#: src/sbackup/ui/restoregui.py:838
#, fuzzy
msgid "Unable to delete snapshot"
msgstr "Echec de la mise à jour de la sauvegarde."

#: src/sbackup/ui/restoregui.py:844
#, python-format
msgid ""
"The selected snapshot '%s' is referenced by more recent snapshots as base "
"snapshot.\n"
"\n"
"Remove all of these child snapshots in order to delete this snapshot."
msgstr ""

#: src/sbackup/ui/restoregui.py:846
#, fuzzy
msgid "Unable to remove snapshot"
msgstr "Echec de la mise à jour de la sauvegarde."

#: src/sbackup/ui/restoregui.py:854
msgid "Termination in progress"
msgstr ""

#: src/sbackup/ui/restoregui.py:870 src/sbackup/ui/restoregui.py:876
#: src/sbackup/ui/restoregui.py:882 src/sbackup/ui/restoregui.py:888
#: data/ui/sbackup-restore.glade:1039
#, fuzzy
msgid "Simple Backup restoration"
msgstr "NSbackup Restauration"

#: src/sbackup/ui/restoregui.py:871 src/sbackup/ui/restoregui.py:877
msgid "<b>Restoring of selected files</b>"
msgstr "Restauration des fichiers sélectionnés"

#: src/sbackup/ui/restoregui.py:872
#, python-format
msgid "Restoring of <tt>'%s'</tt> is in progress."
msgstr "Restauration de <tt>'%s'</tt> en cours"

#: src/sbackup/ui/restoregui.py:873
#, python-format
msgid "Restoring of <tt>'%s'</tt> was successful."
msgstr ""

#: src/sbackup/ui/restoregui.py:874
#, fuzzy, python-format
msgid ""
"Restoring of <tt>'%s'</tt> was not successful.\n"
"\n"
"Following error occurred:\n"
msgstr "Restauration de <tt>'%s'</tt> en cours"

#: src/sbackup/ui/restoregui.py:878
#, python-format
msgid ""
"Restoring of <tt>'%(source)s'</tt>\n"
"to <tt>'%(dirname)s'</tt> is in progress."
msgstr ""
"Restauration de <tt>'%(source)s'</tt>\n"
"dans <tt>'%(dirname)s'</tt> en cours."

#: src/sbackup/ui/restoregui.py:879
#, python-format
msgid ""
"Restoring of <tt>'%(source)s'</tt>\n"
"to <tt>'%(dirname)s'</tt> was successful."
msgstr ""

#: src/sbackup/ui/restoregui.py:880
#, fuzzy, python-format
msgid ""
"Restoring of <tt>'%(source)s'</tt>\n"
"to <tt>'%(dirname)s'</tt> was not successful.\n"
"\n"
"Following error occurred:\n"
msgstr ""
"Restauration de <tt>'%(source)s'</tt>\n"
"dans <tt>'%(dirname)s'</tt> en cours."

#: src/sbackup/ui/restoregui.py:883 src/sbackup/ui/restoregui.py:889
msgid "<b>Reverting selected files</b>"
msgstr "Rétrogradant les fichiers sélectionnés"

#: src/sbackup/ui/restoregui.py:884
#, python-format
msgid "Reverting of <tt>'%s'</tt> is in progress.\n"
msgstr "Rétrogradation de <tt>'%s'</tt> en cours.\n"

#: src/sbackup/ui/restoregui.py:885
#, python-format
msgid "Reverting of <tt>'%s'</tt> was successful."
msgstr ""

#: src/sbackup/ui/restoregui.py:886
#, fuzzy, python-format
msgid ""
"Reverting of <tt>'%s'</tt> was not successful.\n"
"\n"
"Following error occurred:\n"
msgstr "Rétrogradation de <tt>'%s'</tt> en cours.\n"

#: src/sbackup/ui/restoregui.py:890
#, python-format
msgid ""
"Reverting of <tt>'%(source)s'</tt>\n"
"to <tt>'%(dirname)s'</tt> is in progress."
msgstr ""
"Rétrogradation de <tt>'%(source)s'</tt> \n"
"dans <tt>'%(dirname)s'</tt> en cours."

#: src/sbackup/ui/restoregui.py:891
#, python-format
msgid ""
"Reverting of <tt>'%(source)s'</tt>\n"
"to <tt>'%(dirname)s'</tt> was successful."
msgstr ""

#: src/sbackup/ui/restoregui.py:892
#, fuzzy, python-format
msgid ""
"Reverting of <tt>'%(source)s'</tt>\n"
"to <tt>'%(dirname)s'</tt> was not successful.\n"
"\n"
"Following error occurred:\n"
msgstr ""
"Rétrogradation de <tt>'%(source)s'</tt> \n"
"dans <tt>'%(dirname)s'</tt> en cours."

#: src/sbackup/util/__init__.py:439
#, python-format
msgid "%(mb)d MiB %(kb)d KiB"
msgstr ""

#: src/sbackup/util/__init__.py:442
#, python-format
msgid "%(mb)d MiB %(kb)d KiB %(bytes)d"
msgstr ""

#: src/sbackup/util/__init__.py:447
#, python-format
msgid "%(mb)d MB %(kb)d kB"
msgstr ""

#: src/sbackup/util/__init__.py:450
#, python-format
msgid "%(mb)d MB %(kb)d kB %(bytes)d"
msgstr ""

#: src/sbackup/util/filecollect.py:335
#, python-format
msgid "File '%(file)s' does not exist."
msgstr "Le fichier '%(file)s' n'existe pas."

#: src/sbackup/util/filecollect.py:348
#, python-format
msgid "File '%(file)s' is not accessable with error '%(error)s'."
msgstr "Le fichier '%(file)s' n'est pas accessible, erreur '%(error)s'"

#: src/sbackup/util/filecollect.py:364
#, python-format
msgid "File '%(file)s' cannot be opened for read access. Operation timed out."
msgstr ""
"Impossible d'ouvrir le fichier '%(file)s' en lecture. Délai dépassé pour "
"l'opération."

#: src/sbackup/util/filecollect.py:368
#, python-format
msgid ""
"File '%(file)s' cannot be opened for read access with error '%(error)s'."
msgstr ""
"Le fichier '%(file)s' ne peut être ouvert en lecture, erreur '%(error)s'"

#: src/sbackup/util/filecollect.py:385
#, python-format
msgid "File '%(file)s' is backup's target directory."
msgstr ""
"Le fichier '%(file)s' est le répertoire de destination de la sauvegarde."

#: src/sbackup/util/filecollect.py:394
#, python-format
msgid "Symbolic link '%(path)s' -> '%(ln_target)s' is a circular symlink."
msgstr ""
"Le lien symbolique '%(path)s' -> '%(ln_target)s' est un lien symbolique "
"circulaire."

#: src/sbackup/util/filecollect.py:414
#, python-format
msgid ""
"File '%(file)s' exceeds maximum file size ( %(filesize)s > %(maxsize)s)."
msgstr ""
"Le fichier '%(file)s' excède la taille maximale de fichier ( %(filesize)s > %"
"(maxsize)s)."

#: src/sbackup/util/filecollect.py:424
#, python-format
msgid "File '%(file)s' matches regular expression '%(regex)s'."
msgstr "Le fichier '%(file)s' correspond à l'expression régulière '%(regex)s'."

#: src/sbackup/util/filecollect.py:430
#, python-format
msgid "Path '%(file)s' defined in excludes list."
msgstr ""

#: src/sbackup/util/filecollect.py:501
#, python-format
msgid "Error while checking directory '%(dir)s': %(error)s."
msgstr "Erreur lors de la vérification du répertoire '%(dir)s' : %(error)s."

#: src/sbackup/util/filecollect.py:552
msgid "Empty regular expression found. Skipped."
msgstr "Une expression régulière vide a été trouvée et ignorée."

#: src/sbackup/util/filecollect.py:558
#, python-format
msgid "Invalid regular expression ('%s') found. Skipped."
msgstr "L'expression régulière ('%s') invalide a été ignorée."

#: src/sbackup/util/filecollect.py:577
msgid "No directories to backup defined."
msgstr "Aucun répertoire de destination de la sauvegarde n'a été défini."

#: src/sbackup/util/lock.py:76
msgid "Another application instance is already running."
msgstr ""

#: src/sbackup/util/lock.py:78
msgid "Invalid lock file found. Is being removed."
msgstr "Fichier verrou invalide. En cours de suppression."

#: src/sbackup/util/lock.py:128 src/sbackup/util/lock.py:142
#, python-format
msgid "Unable to remove lock file: %s"
msgstr "Impossible de supprimer le fichier verrou : %s"

#: src/sbackup/util/lock.py:132 src/sbackup/util/lock.py:144
#, fuzzy
msgid "Unable to remove lock file: File not found."
msgstr "Impossible de supprimer le fichier verrou : %s"

#: data/ui/sbackup-config.glade:11
msgid "Exclude file type"
msgstr "Exclure un type de fichier"

#: data/ui/sbackup-config.glade:26
msgid "Standard file types:"
msgstr "Types standards de fichiers"

#: data/ui/sbackup-config.glade:63
msgid "Custom defined file type with "
msgstr "Types de fichiers personnalisés définis par "

#: data/ui/sbackup-config.glade:93
msgid "file extension"
msgstr "extension de fichier"

#: data/ui/sbackup-config.glade:163
msgid "Exclude by regex"
msgstr "Exclure par regex (expression régulière)"

#: data/ui/sbackup-config.glade:180
msgid ""
"Enter a regular expression.\n"
"Files and directories matching this expression will be excluded from backup."
msgstr ""
"Entre une expression régulière.\n"
"Les fichiers et les répertoires correspondant à cette expression seront "
"exclus de la sauvegarde."

#: data/ui/sbackup-config.glade:257
msgid "Enter the remote target to include"
msgstr "Entrer une cible distante à inclure"

#: data/ui/sbackup-config.glade:278
msgid "Choose a plugin to get the format"
msgstr "Choisir le plugin pour connaitre le format"

#: data/ui/sbackup-config.glade:295
msgid "Choose your plugin :"
msgstr "Choisir son plugin"

#: data/ui/sbackup-config.glade:367
msgid "Check"
msgstr "Vérifier"

#: data/ui/sbackup-config.glade:479
msgid ""
"<b>Save changed configuration?</b>\n"
"\n"
"The configuration has changed. If you want to use the modified "
"configuration, you have to save it.\n"
"\n"
"Do you want to save the configuration now?"
msgstr ""

#: data/ui/sbackup-config.glade:572
msgid "Configuration Profile Manager"
msgstr ""

#: data/ui/sbackup-config.glade:594
msgid ""
"You can set up multiple configuration profiles which are processed when "
"performing a backup. Each profile has its own set of settings including "
"backup destination, backup format etc."
msgstr ""

#: data/ui/sbackup-config.glade:613
msgid "<b>Configuration profiles</b>"
msgstr "<b>Profils de Configuration</b>"

#: data/ui/sbackup-config.glade:785
msgid ""
"<b>Add new profile</b>\n"
"\n"
"Please enter the new profile name and set the according checkbox in order to "
"enable resp. disable it."
msgstr ""

#: data/ui/sbackup-config.glade:810
msgid "Profile name"
msgstr ""

#: data/ui/sbackup-config.glade:836
msgid "Enabled"
msgstr ""

#: data/ui/sbackup-config.glade:915 data/ui/sbackup-restore.glade:7
#, fuzzy
msgid "Simple Backup Suite"
msgstr "Erreur (Not So) Simple Backup"

#: data/ui/sbackup-config.glade:932
msgid "_File"
msgstr "_Fichier"

#: data/ui/sbackup-config.glade:971
msgid "Save as..."
msgstr ""

#: data/ui/sbackup-config.glade:1008
msgid "_Tools"
msgstr "_Outils"

#: data/ui/sbackup-config.glade:1016 data/ui/sbackup-config.glade:1140
msgid "Set default settings..."
msgstr ""

#: data/ui/sbackup-config.glade:1037
msgid "Profile Manager..."
msgstr ""

#: data/ui/sbackup-config.glade:1058
msgid "Make backup now"
msgstr ""

#: data/ui/sbackup-config.glade:1079
msgid "_Help"
msgstr "_Aide"

#: data/ui/sbackup-config.glade:1087
msgid "_View all help topics"
msgstr ""

#: data/ui/sbackup-config.glade:1103
msgid "_About this program"
msgstr ""

#: data/ui/sbackup-config.glade:1154
msgid "Reload configuration"
msgstr ""

#: data/ui/sbackup-config.glade:1168
msgid "Save configuration"
msgstr ""

#: data/ui/sbackup-config.glade:1191
msgid "Make a backup now"
msgstr ""

#: data/ui/sbackup-config.glade:1214
msgid "Open the profile manager"
msgstr ""

#: data/ui/sbackup-config.glade:1215
msgid "Profiles"
msgstr "Profils"

#: data/ui/sbackup-config.glade:1253
msgid ""
"none\n"
"gzip\n"
"bzip2"
msgstr ""
"aucun\n"
"gzip\n"
"bzip2"

#: data/ui/sbackup-config.glade:1271
msgid ""
"Compression format\n"
"Default is set to uncompressed backups."
msgstr ""

#: data/ui/sbackup-config.glade:1290 data/ui/sbackup-config.glade:1403
msgid ""
"You can split the snapshot to make backup on some filesystem that doesn't "
"support large files. This functionality can not be used if the backup is "
"compressed."
msgstr ""

#: data/ui/sbackup-config.glade:1330
msgid "Do a full backup at least once every"
msgstr "Faire une sauvegarde complète au moins tous les"

#: data/ui/sbackup-config.glade:1339
msgid ""
"Unless a set number of days has passed since the last full backup, "
"incremental backups will be done."
msgstr ""

#: data/ui/sbackup-config.glade:1356 data/ui/sbackup-config.glade:3015
msgid "days"
msgstr "jours"

#: data/ui/sbackup-config.glade:1370
msgid " "
msgstr " "

#: data/ui/sbackup-config.glade:1380 data/ui/sbackup-config.glade:2323
msgid "MiB"
msgstr ""

#: data/ui/sbackup-config.glade:1405
msgid "Split backup archives into several chunks"
msgstr ""

#: data/ui/sbackup-config.glade:1438 data/ui/sbackup-config.glade:2627
#: data/ui/sbackup-config.glade:2863 data/ui/sbackup-config.glade:3111
#: data/ui/sbackup-config.glade:3856
msgid "Help"
msgstr ""

#: data/ui/sbackup-config.glade:1442 data/ui/sbackup-config.glade:2632
msgid "Show help topic about configuration GUI."
msgstr ""

#: data/ui/sbackup-config.glade:1475
msgid "General"
msgstr "Général"

#: data/ui/sbackup-config.glade:1495
msgid "<b>Included files and directories</b>"
msgstr ""

#: data/ui/sbackup-config.glade:1570 data/ui/sbackup-config.glade:1818
#: data/ui/sbackup-config.glade:2023 data/ui/sbackup-config.glade:2181
msgid "Remove"
msgstr ""

#: data/ui/sbackup-config.glade:1620 data/ui/sbackup-config.glade:1868
msgid "Add File"
msgstr "Ajouter un Fichier"

#: data/ui/sbackup-config.glade:1671 data/ui/sbackup-config.glade:1919
msgid "Add Directory"
msgstr "Ajouter un Répertoire"

#: data/ui/sbackup-config.glade:1718
msgid "Include"
msgstr "Inclure"

#: data/ui/sbackup-config.glade:1739
msgid "<b>Excluded files and directories</b>"
msgstr ""

#: data/ui/sbackup-config.glade:1952
msgid "Paths"
msgstr "Chemins"

#: data/ui/sbackup-config.glade:2074 data/ui/sbackup-config.glade:2232
msgid "Add"
msgstr ""

#: data/ui/sbackup-config.glade:2109
msgid "File types"
msgstr "Types de fichiers"

#: data/ui/sbackup-config.glade:2267
msgid ""
"Regular\n"
"Expressions"
msgstr ""

#: data/ui/sbackup-config.glade:2288
msgid "Do not backup files bigger than"
msgstr ""

#: data/ui/sbackup-config.glade:2344
msgid "Follow symbolic links"
msgstr ""

#: data/ui/sbackup-config.glade:2373
msgid "Others"
msgstr "Autres"

#: data/ui/sbackup-config.glade:2400
msgid "Exclude"
msgstr "Exclure"

#: data/ui/sbackup-config.glade:2421
msgid "Use default backup directory"
msgstr ""

#: data/ui/sbackup-config.glade:2446
msgid "/var/backup"
msgstr ""

#: data/ui/sbackup-config.glade:2472
msgid "Use custom local backup directory"
msgstr "Utiliser un répertoire de sauvegarde local personnalisé"

#: data/ui/sbackup-config.glade:2500
msgid "Select the destination folder"
msgstr "Choisir le répertoire de destination"

#: data/ui/sbackup-config.glade:2526
#, fuzzy
msgid "Use a remote site"
msgstr "Ajouter une source distante"

#: data/ui/sbackup-config.glade:2568
msgid "ssh://username:password@example.com/remote/dir/"
msgstr "ssh://nomdutilisateur:motdepasse@exemple.fr/distant/repertoire/"

#: data/ui/sbackup-config.glade:2580
msgid "Connect..."
msgstr ""

#: data/ui/sbackup-config.glade:2672
msgid ""
"<i>The selected destination is not writeable with current permissions. "
"Backups cannot be stored there.</i>"
msgstr ""

#: data/ui/sbackup-config.glade:2704
msgid "Destination"
msgstr "Destination"

#: data/ui/sbackup-config.glade:2725
msgid "_No scheduled backups"
msgstr ""

#: data/ui/sbackup-config.glade:2742
msgid "_Simple"
msgstr ""

#: data/ui/sbackup-config.glade:2747
#, fuzzy
msgid ""
"This option will make a backup not at a precise time but when the computer "
"is up. "
msgstr ""
"Cette option va pousser sbackup à ne pas effectuer les sauvegardes à une "
"heure précise mais quand l'ordinateur est disponible. "

#: data/ui/sbackup-config.glade:2761
msgid "_Custom"
msgstr ""

#: data/ui/sbackup-config.glade:2781
msgid "Do backups"
msgstr ""

#: data/ui/sbackup-config.glade:2798
msgid "Cron time definition"
msgstr ""

#: data/ui/sbackup-config.glade:2834
msgid ""
"hourly\n"
"daily\n"
"weekly\n"
"monthly"
msgstr ""

#: data/ui/sbackup-config.glade:2867
msgid "Show help topic about scheduled backups."
msgstr ""

#: data/ui/sbackup-config.glade:2905
msgid ""
"<small>You can configure scheduled (i.e. automated) backups either simply by "
"choosing an interval or precisely by a custom cron definition. See \"man 5 "
"crontab\" for cron time definition format.</small>"
msgstr ""

#: data/ui/sbackup-config.glade:2937
msgid "Schedule"
msgstr ""

#: data/ui/sbackup-config.glade:2955
#, fuzzy
msgid "Enable purging of old backups"
msgstr "Activer la purge des sauvegardes anciennes ou incomplètes"

#: data/ui/sbackup-config.glade:2983
msgid "Simple cut-off: Erase all backups older than"
msgstr ""

#: data/ui/sbackup-config.glade:3039
msgid "Logarithmic"
msgstr "Logarithmique"

#: data/ui/sbackup-config.glade:3065
msgid ""
"<small>Keep progressively less backups into the past:\n"
"Keep all backups from yesterday.\n"
"Keep one backup per day from last week.\n"
"Keep one backup per week from last month.\n"
"Keep one backup per month from last year.\n"
"Keep one backup per year further into past.\n"
"Erase all other backups.</small>"
msgstr ""

#: data/ui/sbackup-config.glade:3116
msgid "Show help topic about purging of backups."
msgstr ""

#: data/ui/sbackup-config.glade:3152
msgid ""
"<small>Note: Additionally all incomplete and irrecoverable backups will be "
"erased as long as there is a complete backup snapshot made later on.</small>"
msgstr ""
"<small>Note : De plus, toutes les sauvegardes non terminées ou non "
"récupérables seront effacées tant qu'il y a une sauvegarde complète faite "
"plus tard.</small>"

#: data/ui/sbackup-config.glade:3184
msgid "Purging"
msgstr "Nettoyage"

#: data/ui/sbackup-config.glade:3186
msgid "Cleaning old backups"
msgstr "Nettoyage des anciennes sauvegardes"

#: data/ui/sbackup-config.glade:3209
msgid "<b>Logging</b>"
msgstr ""

#: data/ui/sbackup-config.glade:3234
msgid "Debug is just for debbuging purpose.Info is the recommended setting"
msgstr "Debug est juste pour le déboguage.Info est la valeur recommandée"

#: data/ui/sbackup-config.glade:3236
msgid "Log level"
msgstr ""

#: data/ui/sbackup-config.glade:3248
msgid "Debug is just for debbuging purpose. Info is the recommended setting."
msgstr ""

#: data/ui/sbackup-config.glade:3274
msgid "Log file directory"
msgstr "Répertoire du fichier de log"

#: data/ui/sbackup-config.glade:3320
msgid "<b>Report by mail</b>"
msgstr ""

#: data/ui/sbackup-config.glade:3350
msgid "From"
msgstr ""

#: data/ui/sbackup-config.glade:3361
msgid "SMTP server"
msgstr ""

#: data/ui/sbackup-config.glade:3375 data/ui/sbackup-config.glade:4053
msgid "Port"
msgstr ""

#: data/ui/sbackup-config.glade:3391
msgid "To"
msgstr ""

#: data/ui/sbackup-config.glade:3478
msgid "Use a username and password"
msgstr "Utiliser un nom d'utilisateur et un password"

#: data/ui/sbackup-config.glade:3508 data/ui/sbackup-config.glade:4075
msgid "Username"
msgstr "Nom d'utilisateur :"

#: data/ui/sbackup-config.glade:3540 data/ui/sbackup-config.glade:4124
msgid "Password"
msgstr "Mot de passe :"

#: data/ui/sbackup-config.glade:3584
msgid "Use secure connection"
msgstr ""

#: data/ui/sbackup-config.glade:3615
msgid "Choose a certificate file"
msgstr ""

#: data/ui/sbackup-config.glade:3629
msgid "Key"
msgstr "Clé"

#: data/ui/sbackup-config.glade:3645
msgid "Certificate"
msgstr "Certificat"

#: data/ui/sbackup-config.glade:3659
msgid "Choose a key file"
msgstr ""

#: data/ui/sbackup-config.glade:3731
msgid "Test _mail settings"
msgstr ""

#: data/ui/sbackup-config.glade:3772
msgid "Report"
msgstr "Rapport"

#: data/ui/sbackup-config.glade:3811
msgid "Warning"
msgstr ""

#: data/ui/sbackup-config.glade:3860
msgid "Show help topic about default values."
msgstr ""

#: data/ui/sbackup-config.glade:3893
msgid "_Set"
msgstr ""

#: data/ui/sbackup-config.glade:3935
#, fuzzy
msgid "Set remote target"
msgstr "Entrer une cible distante à inclure"

#: data/ui/sbackup-config.glade:3957
msgid "Type of service"
msgstr ""

#: data/ui/sbackup-config.glade:3997
#, fuzzy
msgid "Hostname"
msgstr "Nom d'utilisateur :"

#: data/ui/sbackup-config.glade:4027
#, fuzzy
msgid "<b>Details</b>"
msgstr "<b>Détails de la Sauvegarde</b>"

#: data/ui/sbackup-config.glade:4157
#, fuzzy
msgid "Show password"
msgstr "Mot de passe :"

#: data/ui/sbackup-config.glade:4217
#, fuzzy
msgid "_Connect"
msgstr "Contenu"

#: data/ui/sbackup-config.glade:4293
msgid "Make a full backup"
msgstr ""

#: data/ui/sbackup-restore.glade:38
#, fuzzy
msgid "<b>Backup destination</b>"
msgstr "NSbackup Restauration"

#: data/ui/sbackup-restore.glade:52
msgid "Displays the manual in Help browser."
msgstr ""

#: data/ui/sbackup-restore.glade:85
#, fuzzy
msgid "Default profile's destination"
msgstr "Profil par défaut"

#: data/ui/sbackup-restore.glade:90
msgid ""
"Set backup destination to the place specified in default configuration "
"profile"
msgstr ""

#: data/ui/sbackup-restore.glade:137
#, fuzzy
msgid "Custom destination"
msgstr "Destination"

#: data/ui/sbackup-restore.glade:143
msgid ""
"Select a backup destination on your harddisk or at a remote site. Press "
"Apply when ready."
msgstr ""

#: data/ui/sbackup-restore.glade:184
msgid "Choose a destination folder. For remote sites fill URI in text field"
msgstr ""

#: data/ui/sbackup-restore.glade:201
msgid "Change destination to path specified in left text field"
msgstr ""

#: data/ui/sbackup-restore.glade:266
msgid "Select a bold printed date to get the list of backups for that date."
msgstr ""

#: data/ui/sbackup-restore.glade:310
#, fuzzy
msgid "Available backups"
msgstr "<b>Sauvegardes disponibles</b>"

#: data/ui/sbackup-restore.glade:398
#, fuzzy
msgid "Delete snapshot"
msgstr "Supprimer la sauvegarde courante?"

#: data/ui/sbackup-restore.glade:436
msgid "Delete"
msgstr "Supprimer"

#: data/ui/sbackup-restore.glade:475
#, fuzzy
msgid "Upgrade snapshot"
msgstr "Echec de la mise à jour de la sauvegarde."

#: data/ui/sbackup-restore.glade:515
msgid "Upgrade"
msgstr "Mettre à jour"

#: data/ui/sbackup-restore.glade:553
msgid "<b>Snapshot History</b>"
msgstr "<b>Historique des sauvegardes</b>"

#: data/ui/sbackup-restore.glade:579
#, fuzzy
msgid "Current base:"
msgstr "Profile courant : %s"

#: data/ui/sbackup-restore.glade:620
msgid "Select a snapshot on this list to rebase on it."
msgstr "Sélectionner la sauvegarde dans la liste pour l'appliquer comme base."

#: data/ui/sbackup-restore.glade:695
msgid "Rebase"
msgstr "Changer la base"

#: data/ui/sbackup-restore.glade:746
msgid "<b>Snapshot Management</b>"
msgstr "<b>Gestion de sauvegarde</b>"

#: data/ui/sbackup-restore.glade:785
#, fuzzy
msgid "Snapshot details"
msgstr "<b>Détails de la Sauvegarde</b>"

#: data/ui/sbackup-restore.glade:850 data/ui/sbackup-restore.glade:867
msgid "Restores files/directories from the selected snapshot"
msgstr ""

#: data/ui/sbackup-restore.glade:862
msgid "Restore"
msgstr "restituer"

#: data/ui/sbackup-restore.glade:882 data/ui/sbackup-restore.glade:901
msgid ""
"Restores files/directories recursively starting from the base snapshot to "
"the selected snapshot"
msgstr ""

#: data/ui/sbackup-restore.glade:896
msgid "Revert"
msgstr "Rétrograder"

#: data/ui/sbackup-restore.glade:939
msgid "<b>Restoration Management</b>"
msgstr "<b>Gestion de la restoration</b>"

#: data/ui/sbackup-restore.glade:964
msgid "<b>Export Management</b>"
msgstr "<b>Gestion de l'exportation</b>"

#: data/ui/sbackup-restore.glade:1056
msgid "Headline"
msgstr "Titre"

#: data/ui/sbackup-restore.glade:1069
msgid "Content"
msgstr "Contenu"

#: scripts/sbackup-upgrade-backups.py:45
msgid ""
"\n"
"Simple Backup suit command line backup format upgrade\n"
"Usage: upgrade-backup backup-target-url\n"
"Note: backup-target-url must not include the snapshot subdirectory name, for "
"example:\n"
"\n"
"   /var/backup/\n"
"\n"
"Use simple-restore-gnome for more ease of use.\n"
msgstr ""
"\n"
"Utilitaire en ligne de commande de mise à jour du format de sauvegarde de la "
"suite Simple Backup\n"
"Usage : upgrade-backup chemin-de-sauvegarde-cible\n"
"Note : chemin-de-sauvegarde-cible ne doit pas inclure de sous-répertoire de "
"sauvegarde précise, par exemple :\n"
"\n"
"   /var/sauvegarde/\n"
"\n"
"Utilisez simple-restore-gnome pour plus de facilité d'utilisation.\n"

#, fuzzy
#~ msgid "An error occurred during the backup: %s"
#~ msgstr ""
#~ "Une erreur s'est produite durant la sauvegarde :\n"
#~ "%s"

#~ msgid "(Not So) Simple Backup error"
#~ msgstr "Erreur (Not So) Simple Backup"

#~ msgid "Import of module 'pynotify' failed with error: %s."
#~ msgstr "L'import du module 'pynotify' a échoué avec l'erreur : %s."

#~ msgid "Failed to set the gid to 'admin' one :"
#~ msgstr "Echec du changement du groupe ID à 'admin'"

#~ msgid "Setting split size."
#~ msgstr "Réglage de la taille de découpe."

#~ msgid "Preparation of backup is done. Archive is being created."
#~ msgstr ""
#~ "La préparation de la sauvegarde est terminée. L'archive est en cours de "
#~ "création."

#~ msgid "Target not writeable: "
#~ msgstr "Il n'est pas permis d'écrire dans le répertoire de destination. "

#~ msgid "Invalid value of the target directory : "
#~ msgstr "Valeur du répertoire de destination des sauvegardes invalide "

#~ msgid ""
#~ "Got a non valid snapshot '%(name)s' due to name convention : %"
#~ "(error_cause)s "
#~ msgstr ""
#~ "Sauvegarde '%(name)s' non valide à cause de la convention de nommage : %"
#~ "(error_cause)s "

#~ msgid "Deleting snapshot: '%(snp)s'."
#~ msgstr "Suppression de l'instantané : '%(snp)s'."

#~ msgid "Cannot downgrade other format than 'gzip' to 1.4"
#~ msgstr "Impossible de baiser le niveau d'un format autre que 'gzip' à 1.4"

#~ msgid ""
#~ "A backup run is initiated in the background.\n"
#~ "The process id is: %s."
#~ msgstr ""
#~ "Une sauvegarde est en cours en tâche de fond.\n"
#~ "Le numéro de processus est : %s."

#~ msgid "Test on '%s' didn't succeed!"
#~ msgstr "Le test sur '%s' a échoué !"

#~ msgid "'%d' test(s) didn't succeed !"
#~ msgstr "'%d' tests n'ont pas reussi !"

#~ msgid ""
#~ "Please test writability of the target directory by pressing \"Test\" "
#~ "button on the right."
#~ msgstr ""
#~ "Veuillez tester la capacité d'écriture sur le répertoire cible en "
#~ "appuyant sur le bouton \"Test\" sur la droite."

#~ msgid "Test Succeeded !"
#~ msgstr "Test réussi"

#~ msgid "Target directory is writable."
#~ msgstr "Le répertoire cible n'est pas inscriptible."

#~ msgid ""
#~ "Please change target directory and test writability of the target "
#~ "directory by pressing \"Test\" button on the right."
#~ msgstr ""
#~ "Veuillez changer le répertoire de destination et tester les permissions "
#~ "d'écriture en appuyant sur le bouton \"Test\" à droite."

#~ msgid "disable"
#~ msgstr "désactiver"

#~ msgid ""
#~ "You can't %s the Default Profile. Please use it if you need only one "
#~ "profile."
#~ msgstr ""
#~ "Vous ne pouvez pas %s le Profil Par Défaut. Merci de l'utiliser si vous "
#~ "n'avez besoin que d'un profil."

#~ msgid ""
#~ "The program is going to be terminated. Please make sure the missing "
#~ "directory exists (e.g. by mounting an external disk) or change the "
#~ "specified target in '(Not So) Simple Backup-Configuration' tool and "
#~ "restart this application."
#~ msgstr ""
#~ "Le programme va se terminer. Merci de vérifier que le répertoire manquant "
#~ "existe (par exemple en montant un disque externe) ou en modifier la "
#~ "destination spécifiée dans l'outil '(Not So) Simple Backup-Configuration' "
#~ "et redémarrer l'application."

#~ msgid "An error occurred during initialization:"
#~ msgstr "Une erreur s'est produite pendant l'initialisation :"

#~ msgid "Actual base : <b>%s</b>"
#~ msgstr "Parent actuel : <b>%s</b>"

#~ msgid ""
#~ "Choose the compression format that you want to apply.\n"
#~ "Default is <b>gzip</b>."
#~ msgstr ""
#~ "Choisissez le format de compression que vous voulez appliquer.\n"
#~ "Par défaut <b>gzip</b>"

#~ msgid "Use a remote directory (SSH or FTP)"
#~ msgstr "Utiliser un répertoire distant (SSH ou FTP)"

#~ msgid "Test"
#~ msgstr "Tester"

#~ msgid ""
#~ "Select the directory in which you have your backups.\n"
#~ "You can use a <b>local</b> or <b>distant</b> directory depending on the "
#~ "plugins you have installed."
#~ msgstr ""
#~ "Sélectionner le répertoire dans lequel vous avez vos sauvegardes.\n"
#~ "Vous pouvez utiliser un répertoire local où distant selon les modules que "
#~ "vous avez installés."

#~ msgid "Use the default directory : "
#~ msgstr "Utiliser le répertoire par défaut : "

#~ msgid "Use a custom directory : "
#~ msgstr "Utiliser une répertoire personalisé : "

#~ msgid "<b>Snapshots directory</b>"
#~ msgstr "<b>Répertoire de sauvegarde</b>"

#~ msgid "Restore As"
#~ msgstr "Restaurer sous"

#~ msgid "Revert As"
#~ msgstr "Revenir à"

#~ msgid "Choose to upgrade your snapshot if needed."
#~ msgstr "Choisir de mettre à jour votre sauvegarde si besoin est."

#~ msgid "Actual base :"
#~ msgstr "Base actuelle:"

#~ msgid "Select a snapshot and change its base to another snapshot."
#~ msgstr ""
#~ "Sélectionner une sauvegarde et changer sa base à une autre sauvegarde."

#~ msgid "Initializing FUSE FILE ACCESS MANAGER !"
#~ msgstr "Initialisation de FUSE FILE ACCESS MANAGER"

#~ msgid "BackupManager created "
#~ msgstr "Création du BackupManager "

#~ msgid "I didn't find the log file. Please set it up in sbackup.conf "
#~ msgstr ""
#~ "Je n'ai pas pu déterminer l'emplacement du fichier journal. Veuillez le "
#~ "configurer dans sbackup.conf "

#~ msgid "Starting snapshot %(name)s "
#~ msgstr "Début de la sauvegarde %(name)s "

#~ msgid ""
#~ "We don't have read access to '%(file)s', it has to be exclude : %(error)s "
#~ msgstr ""
#~ "Nous n'avons pas accès en lecture à '%(file)s'. Il doit être exclu : %"
#~ "(error)s "

#~ msgid "'%(file)s' doesn't exist, it has to be exclude "
#~ msgstr "'%(file)s' n'existe pas. Il doit être exclu "

#~ msgid "Problem with '%(file)s' : %(error)s "
#~ msgstr "Un problem est survenu avec '%(file)s' : %(error)s "

#~ msgid "got an error with '%(file)s' : %(error)s"
#~ msgstr "Une erreur est survenue avec  '%(file)s' : %(error)s"

#~ msgid ""
#~ "'%(file)s' size is higher than the specified one ( %(filesize)s > %"
#~ "(maxsize)s), it has to be exclude "
#~ msgstr ""
#~ "La taille de '%(file)s' est supérieure à celle définie ( %(filesize)s > %"
#~ "(maxsize)s), Il doit être exclu. "

#~ msgid "Terminating FUSE FILE ACCESS MANAGER !"
#~ msgstr "Clôture de FUSE FILE ACCESS MANAGER !"

#~ msgid "Set the reciever of this mail"
#~ msgstr "Destinataire du message"

#~ msgid "Choose the SSL option (smtptls=1) to be able to use Cert and Key"
#~ msgstr ""
#~ "Choisissez l'option SSL (smtptls=1) pour être capable d'utiliser un "
#~ "Certificat et une Clé."

#~ msgid ""
#~ "When setting a username (resp password), password (resp username) is "
#~ "mandatory"
#~ msgstr ""
#~ "Si vous spécifiez un nom d'utilisateur (resp. mot de passe), vous devez "
#~ "aussi spécifier un mot de passe (resp nom d'utilisateur)"

#~ msgid "Save to file ..."
#~ msgstr "Enregistrer sous le fichier ..."

#~ msgid "Use custom backup settings"
#~ msgstr "Utiliser des paramètres de sauvegarde personnalisés"

#~ msgid "Use recommended backup settings"
#~ msgstr "Utiliser les paramètres de sauvegarde recommandés"

#~ msgid "Test all"
#~ msgstr "Tester Tout"

#~ msgid ""
#~ "<small>Do backups according to the settings defined on the next pages of "
#~ "this dialog.</small>"
#~ msgstr ""
#~ "<small>Effectuer les sauvegardes selon les paramètres définis dans les "
#~ "pages suivantes de cette fenêtre de dialogue</small>"

#~ msgid ""
#~ "<small>Do not backup your data automatically. Note: this does not prevent "
#~ "doing manual backups.</small>"
#~ msgstr ""
#~ "<small>Ne pas sauvegarder automatiquement vos données. Notez bien: ceci "
#~ "ne vous empêche pas de faire des sauvegardes manuelles.</small>"

#~ msgid "Mb"
#~ msgstr "Mo"

#~ msgid "Regex"
#~ msgstr "Regex (expression régulière)"

#~ msgid "Use default backup directory (/var/backup)"
#~ msgstr "Utiliser le répertoire de sauvegarde par défaut (/var/backup)"

#~ msgid "simply"
#~ msgstr "simplement"

#~ msgid "precisely"
#~ msgstr "précisément"

#~ msgid "at:"
#~ msgstr "à:"

#~ msgid "Minute"
#~ msgstr "Minute"

#~ msgid "Hour"
#~ msgstr "Heure"

#~ msgid "Day of week:"
#~ msgstr "Jour de la semaine:"

#~ msgid "Day of month:"
#~ msgstr "Jour du mois:"

#~ msgid "Log settings"
#~ msgstr "Paramétrages des logs"

#~ msgid ""
#~ "Debug is just for debbuging purpose\n"
#~ "Info is the recommended setting"
#~ msgstr ""
#~ "Debug est juste pour le déboguage\n"
#~ "Info est la valeur recommandée"

#~ msgid "Parameters"
#~ msgstr "Paramètres"

#~ msgid "Security"
#~ msgstr "Sécurité"

#~ msgid "Time (for admins only)"
#~ msgstr "Plannification (seulement pour les admins)"

#~ msgid "Got an exception when Pulling '%s' : "
#~ msgstr "Erreur récupérée en transférant '%s' "

#~ msgid "Snapshot '%s' is a full . Can't rebase on older snapshot"
#~ msgstr ""
#~ "La sauvegarde '%s' est une sauvegarde complète. Nous ne pouvons changer "
#~ "la base sur une plus ancienne sauvegarde."

#~ msgid "Target '%s' directory is excluded"
#~ msgstr "Le répertoire de destination '%s' est exclue"

#~ msgid "Deleting '%(snp)s' for purge !"
#~ msgstr "Suppression de '%(snp)s' pour la purge !"

#~ msgid "Non existing directory : %s"
#~ msgstr "Dossier non existant : %s"

#~ msgid "gtk-no"
#~ msgstr "Non"

#~ msgid "gtk-yes"
#~ msgstr "Oui"

#~ msgid "gtk-remove"
#~ msgstr "gtk-remove"

#~ msgid "gtk-add"
#~ msgstr "gtk-add"

#~ msgid "gtk-close"
#~ msgstr "gtk-close"

#~ msgid "gtk-ok"
#~ msgstr "gtk-ok"

#~ msgid "gtk-cancel"
#~ msgstr "gtk-cancel"

#~ msgid "gtk-edit"
#~ msgstr "gtk-edit"

#~ msgid "Unable to change permissions for "
#~ msgstr "Impossible de changer les permissions pour "

#~ msgid "sbackup configuration error"
#~ msgstr "Erreur de configuration de sbackup"

#~ msgid ""
#~ "Please enter the new profile name in the <b>Profile name</b> entry and "
#~ "check the <b>Enable</b> checkbox to enable it."
#~ msgstr ""
#~ "Entrez le nom du nouveau profile dans l'entrée <b>Nom du profile</b> et "
#~ "cochez la case <b>Activé</b> pour l'activer."

#~ msgid "No directory to backup !"
#~ msgstr "Aucun répertoire à sauvegarder !"

#~ msgid "This is a user friendly backup solution for common desktop needs."
#~ msgstr ""
#~ "Ceci est une solution de sauvegarde conviviale adaptée aux besoins des "
#~ "ordinateurs de bureau."

#~ msgid "File list ready, committing to disk."
#~ msgstr "La liste des fichiers est prête, écriture sur le disque."
